src/main/groovy/com/spidasoftware/schema/conversion/changeset/ConverterUtils.groovy

```diff
@@ -79,6 +79,7 @@ class ConverterUtils {
         converter.addChangeSet(11, new WireStateLabelChangeSet())
         converter.addChangeSet(11, new PoleCutTopChangeSet())
         converter.addChangeSet(11, new MomentAtHeightChangeSet())
+        converter.addChangeSet(11, new ClientTrussChangeSet())
```

Comment:
I think we might want this name to be less specific so we can put the Truss abstract item downconversion in the same changeset.

Reply:
is `TrussChangeSet` better?